src/test/kotlin/pl/newbies/auth/GithubAuthTest.kt

```diff
@@ -0,0 +1,79 @@
+package pl.newbies.auth
+
+import io.ktor.client.plugins.ClientRequestException
+import io.ktor.client.request.forms.submitForm
+import io.ktor.client.request.get
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.Parameters
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import pl.newbies.util.IntegrationTest
+import pl.newbies.util.TestData
+import pl.newbies.util.httpClient
+import pl.newbies.util.loginAs
+
+class GithubAuthTest : IntegrationTest() {
+
+    @Nested
+    inner class Login {
+        @Test
+        fun `should redirect when login called`() = withAres {
+            // when
+            val response = httpClient.get("/oauth/login/github")
+
+            // then
+            assertEquals(HttpStatusCode.OK, response.status)
```

Comment:
*where* redirect check 
it looks like you are only checking if the request is executing, not how

Reply:
Github has a mocked module, which returns "ok" as a content on expected (see line below and githubModule() in TestModules.kt)